test/src/schedule_parse_test.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:cron/src/schedule_parse_exception.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Test throw without message', () {
```

Comment:
I'm not sure if this is something worth of testing.

I'd rather see tests along the lines of using matchers like `throwsA(isA<FormatException>())`, while using the parser. wdyt?

Reply:
Exactly @isoos :) It would be better to test the behavior when parsing actually fails than this test.